src/Annotator.scss

```diff
@@ -467,7 +467,9 @@ $tablet: 'max-width: 768px';
     .bp-btn-annotate-draw-delete,
     .bp-btn-annotate-draw-delete:active,
     .bp-btn-annotate-draw-delete:hover {
-        padding-left: 4px;
+        font-size: 13px;
+        margin-top: -3px;
```

Comment:
I know this isn't great but i haven't been able to figure out a better way of getting from this
<img width="221" alt="screen shot 2017-12-13 at 9 56 23 pm" src="https://user-images.githubusercontent.com/3299001/33977644-b4c96084-e050-11e7-8357-1d11c8a89ac3.png">
to this
<img width="246" alt="screen shot 2017-12-13 at 9 56 12 pm" src="https://user-images.githubusercontent.com/3299001/33977652-bc7260c4-e050-11e7-8a3f-833662918f40.png">

100% open to ideas here!

Reply:
What's the issue? You just want bigger font? Also, can you put a pipe to the left of the X, so we can have a little visual break, in that? 

IE) Sumedha Pramod drew | X Delete

Reply:
it's misaligned. I'm fine keeping the font smaller but the misalignment was the real problem. Not entirely a huge fan of doing a negative margin :-/ 

We removed the pipe between the buttons and the dialog label a while back intentionally. I'll see if I can figure out why we did that.

Reply:
OH I can see the misalignment, now. I think the bigger font and margin drop looks good to me :)

Reply:
👍